### PR TITLE
create_database_issue#3

### PR DESCRIPTION
**Pull request by Twigwalker**
issue #3 
npm install package Jason

### CODE REVIEW

**Comment by FlorfK** [comment]
I see no folder

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "database",
+  "version": "1.0.0",
+  "description": "Weer zo'n leipe test opdracht",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
```
Comment (Wiebehd) [inline package.json]: Dit ziet er goed uit

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "database",
+  "version": "1.0.0",
+  "description": "Weer zo'n leipe test opdracht",
```
Comment (FlorfK) [inline package.json]: huh?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (FlorfK) [inline package.json]: No database folder you're working in the root I think
The root is on fire
Daroot 
I'm rooting for you

**Pull request merged**